docs/new_project_guide.md

```diff
@@ -54,10 +54,12 @@ which includes crash reports, fuzzer statistics, etc and are auto-cced on newly
 tracker.
 * `sanitizers` (optional) - List of sanitizers to use. By default, you shouldn't override this and it 
 will use the default list of supported sanitizers (currently -
-AddressSanitizer("address"), UndefinedBehaviorSanitizer("undefined")). 
+AddressSanitizer("address"), UndefinedBehaviorSanitizer("undefined")).
 If your project does not build with a particular sanitizer configuration and you need some time fixing
 it, then you can use this option to override the defaults temporarily. E.g. For disabling 
 UndefinedBehaviourSanitizer build, then you can just specify all supported sanitizers, except "undefined".
+* `help_url` (optional) - Link to a custom help URL in bug reports instead of the [default OSS-Fuzz guide to reproducing
```

Comment:
custom help URL for ? can you add some words on purpose.

Reply:
Done